apps/web/src/services/asset-inventory/collector/collector-create/modules/CreateCollectorStep4.vue

```diff
@@ -150,17 +150,24 @@ const handleConfirmCreateCollector = async () => {
             collector_id: state.createdCollectorId,
         });
         showSuccessMessage(i18n.t('INVENTORY.COLLECTOR.CREATE.ALT_S_COLLECT_EXECUTION'), '');
-        handleRouteToCollectorList();
+        handleRouteToCollectorDetailPage();
```

Comment:
suggestion: We decided not to use event handler in other function!
But we decided to allow the case of using normal function as a event handler.
So, what about to change the name of function `handleRouteToCollectorDetailPage` to `goToCollectorDetailPage`?

Reply:
WT... 
👍